### ISSUE TITLE
Angular Material 1.1.0-rc1 md-card conflicts with Google Maps 

### ISSUE THREAD

**Issue by GoulwenBZH** [angular/material#7631]
Hi!
Angular Material 1.1.0-rc1 md-card conflicts with Google Maps because of the height 100% important tag.

md-card > :not(md-card-content) img {
    height: 100% !important;
}

A codepen exemple here: http://codepen.io/GoulwenBZH/pen/MyJPqb

**Comment by himanshuapril1** [comment]
Just wrap up your `map-canvas` div into `md-card-content` element, it'll work fine.
Or I've already done it below for you. :) 

```
<md-card>
  <md-card-content>
    <div class="angular-google-map">
      <div id="map-canvas" style="height:350px"></div>
    </div>
  </md-card-content>
</md-card>
```

**Comment by himanshuapril1** [comment]
Closes angular/material#7631

**Comment by DevVersion** [comment]
Not an issue. Thank you @himanshuapril1 for the fix!

Here in the docs is a section, which shows the correct element structure of a `md-card``
- https://material.angularjs.org/HEAD/api/directive/mdCard

Demo with Himanshu's fix: http://codepen.io/DevVersion/pen/wGJJpp
